src/index.js

```diff
@@ -1,18 +1,93 @@
 import './styles.css';
 
+const taskInputBtn = document.querySelector('#taskInputbtn');
 const taskList = document.querySelector('#task-list');
+const taskInput = document.querySelector('#add-task');
+let userArray = [];
+const objStr = localStorage.getItem('Tasks');
+if (objStr != null) {
+  userArray = JSON.parse(objStr);
+}
 
-const arr = [{
-  description: 'wash the dishes',
-  completed: 'false',
-  index: '0',
-}, {
-  description: 'complete to do list project',
-  completed: 'false',
-  index: '1',
-}];
-
-arr.forEach((items) => {
-  taskList.innerHTML += `
-    <div class="tasks"><input type="checkbox" name="first" value="first" id="first"><label class="tasks-text" for="first">${items.description}</label><span class="tasks-icon"><i class="fa fa-ellipsis-v" aria-hidden="true"></i></span></div>`;
+function removetask(removeiconId) {
+  removeiconId.forEach((el, i) => {
+    el.addEventListener('click', () => {
+      const newArray = JSON.parse(localStorage.getItem('Tasks'));
+      newArray.splice(i, 1);
+      localStorage.setItem('Tasks', JSON.stringify(newArray));
+      location.reload(); // eslint-disable-line
+    });
+  });
+}
+function savelocal(text, i) {
+  const newArray = JSON.parse(localStorage.getItem('Tasks'));
+  newArray[i].description = text;
+  localStorage.setItem('Tasks', JSON.stringify(newArray));
+}
+
+function editTask() {
+  const removeiconId = document.querySelectorAll('#remove');
+  const doticon = document.querySelectorAll('#tasks-icon');
+  const textareaInput = document.querySelectorAll('.tasks-text');
+  const taskdiv = document.querySelectorAll('.tasks');
+
+  taskdiv.forEach((el, i) => {
+    el.addEventListener('dblclick', (e) => {
+      e.target.disabled = false;
+      doticon[i].style.display = 'none';
+      removeiconId[i].style.display = 'block';
+      removetask(removeiconId);
+      el.addEventListener('dblclick', (e) => {
+        e.target.disabled = true;
+        doticon[i].style.display = 'block';
+        removeiconId[i].style.display = 'none';
+        editTask();
+        savelocal(textareaInput[i].value, i);
+      });
+    });
+  });
+}
```

Comment:
- [ ] Based on your UI I am unable to edit a task. I think for better user experience when clicking enter or outside the box, the task should get updated/edited. BTW your edit code looks fine :+1: https://www.awesomescreenshot.com/video/16122594?key=560650414e9a64331c8ec122ac688ddc